src/client/tools/playerNameTools.ts

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {
+  uniqueNamesGenerator,
+  adjectives, colors, names,
+} from 'unique-names-generator';
+
+const LOCALSTORAGE_NAME = 'sueca-username';
+
+export const playerNameTools = {
+  get: () => {
+    if (typeof localStorage === 'undefined') {
+      return '';
+    }
+
+    let name = (
+      (typeof localStorage !== 'undefined' ? localStorage.getItem(LOCALSTORAGE_NAME) : '')
+      || ''
+    ).trim();
+
+    if (!name) {
+      name = uniqueNamesGenerator({
+        dictionaries: [adjectives, colors, names],
```

Comment:
We could in the future add our custom dictionary here with some cool words like bino, candido...